src/mango/src/mango_cursor.erl

```diff
@@ -124,6 +126,12 @@ remove_indexes_with_partial_filter_selector(Indexes) ->
     lists:filter(FiltFun, Indexes).
 
 
+remove_unbuilt_indexes(Indexes) ->
```

Comment:
That just means there are now two things in the wrong place. It should move to mango_idx.erl since its an index function. We can move the other in a separate clean up PR later.